src/components/common/YourCompany/AssociationType.vue

```diff
@@ -144,31 +143,36 @@ export default class AssociationType extends Mixins(CommonMixin, EnumMixin) {
   /** Select options */
   readonly associationTypeOptions: Array<any> = [
     {
-      value: AssociationTypes.COMMUNITY_SERVICE_COOPERATIVE,
-      text: this.associationTypeToDescription(AssociationTypes.COMMUNITY_SERVICE_COOPERATIVE)
+      value: CoopTypes.COMMUNITY_SERVICE_COOPERATIVE,
```

Comment:
AssociationType applies to SP GP and CP

CoopType just CP

Reply:
The 3 values all apply to cooperatives, so ...? Should I change something here?